HLeithner/joomla-cms
# Prepared plg content joomla

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Use the plugin in all ways you can think of.

* Create an article in a category
* try to delete the category, you should get an error that the category is not empty
* create an article in the frontend, you should get a e-mail notification



### Expected result
Nothing changed.
